### DISCUSSION TITLE
MSB8036: The Windows SDK version 10.0.22000.0 was not found.

### DISCUSSION THREAD

**Discussion by qinqin123555** [microsoft/vscode-discussions#1129] [Extension Development QnA]
Hello, please help me check. After downloading frontend dependencies using vscode and npm, I'm encountering the error MSB8036: The Windows SDK version 10.0.22000.0 was not found. Below is the version of my vscode configuration: 1.88.0 (user setup)
Commit: 5c3e652f63e798a5ac2f31ffd0d863669328dc4c
Date: 2024-04-03T13:26:18.741Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22621
However, I have already downloaded Windows SDK 10.0.22000.0 locally, and I have restarted the computer. After redownloading vscode, the configuration and error remain the same.

![image](https://github.com/microsoft/vscode-discussions/assets/144591113/b0dac892-590c-4f32-a0c2-f63fe8455276)

npm ERR! code 1
npm ERR! path D:\BaiduNetdiskDownload\数据中台\9-springboot springcloud一站式数据中台 大数据\源码+资料+文档\datax-ui-pro\node_modules\deasync
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node ./build.js
npm ERR! C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 10.0.22000.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [D:\BaiduNetdiskDownload\������̨\9-springboot springcloudһվʽ������̨ ������\Դ��+����+�ĵ�\datax-ui-pro\node_modules\deasync\build\ \deasync.vcxproj]
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@10.0.1
npm ERR! gyp info using node@18.17.1 | win32 | x64
npm ERR! gyp info find Python using Python version 3.9.5 found at "D:\Download\Python\python.exe"
npm ERR! gyp info find VS using VS2022 (17.8.34330.188) found at:
npm ERR! gyp info find VS "C:\Program Files\Microsoft Visual Studio\2022\Community"
npm ERR! gyp info find VS run with --verbose for detailed information
npm ERR! gyp info spawn D:\Download\Python\python.exe
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args 'D:\\Download\\nodejs\\node_global\\node_modules\\npm\\node_modules\\node-gyp\\gyp\\gyp_main.py',
npm ERR! gyp info spawn args 'binding.gyp',
npm ERR! gyp info spawn args '-f',
npm ERR! gyp info spawn args 'msvs',
npm ERR! gyp info spawn args '-I',
npm ERR! gyp info spawn args 'D:\\BaiduNetdiskDownload\\数据中台\\9-springboot springcloud一站式数据中台 大数据\\源码+资料+文档\\datax-ui-pro\\node_modules\\deasync\\build\\config.gypi',
npm ERR! gyp info spawn args '-I',
npm ERR! gyp info spawn args 'D:\\Download\\nodejs\\node_global\\node_modules\\npm\\node_modules\\node-gyp\\addon.gypi',
npm ERR! gyp info spawn args '-I',
npm ERR! gyp info spawn args 'C:\\Users\\qin\\AppData\\Local\\node-gyp\\Cache\\18.17.1\\include\\node\\common.gypi',
npm ERR! gyp info spawn args '-Dlibrary=shared_library',
npm ERR! gyp info spawn args '-Dvisibility=default',
npm ERR! gyp info spawn args '-Dnode_root_dir=C:\\Users\\qin\\AppData\\Local\\node-gyp\\Cache\\18.17.1',
npm ERR! gyp info spawn args '-Dnode_gyp_dir=D:\\Download\\nodejs\\node_global\\node_modules\\npm\\node_modules\\node-gyp',
npm ERR! gyp info spawn args '-Dnode_lib_file=C:\\\\Users\\\\qin\\\\AppData\\\\Local\\\\node-gyp\\\\Cache\\\\18.17.1\\\\<(target_arch)\\\\node.lib',
npm ERR! gyp info spawn args '-Dmodule_root_dir=D:\\BaiduNetdiskDownload\\数据中台\\9-springboot springcloud一站式数据中台 大数据\\源码+资料+文档\\datax-ui-pro\\node_modules\\deasync',
npm ERR! gyp info spawn args '-Dnode_engine=v8',
npm ERR! gyp info spawn args '--depth=.',
npm ERR! gyp info spawn args '--no-parallel',
npm ERR! gyp info spawn args '--generator-output',
npm ERR! gyp info spawn args 'D:\\BaiduNetdiskDownload\\数据中台\\9-springboot springcloud一站式数据中台 大数据\\源码+资料+文档\\datax-ui-pro\\node_modules\\deasync\\build',
npm ERR! gyp info spawn args '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\MSBuild.exe
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args 'build\\binding.sln',
npm ERR! gyp info spawn args '/clp:Verbosity=minimal',
npm ERR! gyp info spawn args '/nologo',
npm ERR! gyp info spawn args '/p:Configuration=Release;Platform=x64'
npm ERR! gyp info spawn args ]
npm ERR! gyp ERR! build error
npm ERR! gyp ERR! stack Error: `C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\MSBuild.exe` failed with exit code: 1
npm ERR! gyp ERR! stack at ChildProcess.<anonymous> (D:\Download\nodejs\node_global\node_modules\npm\node_modules\node-gyp\lib\build.js:209:23)
npm ERR! gyp ERR! stack at ChildProcess.emit (node:events:514:28)
npm ERR! gyp ERR! stack at ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Windows_NT 10.0.22621
npm ERR! gyp ERR! command "D:\\Download\\nodejs\\node.exe" "D:\\Download\\nodejs\\node_global\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"npm ERR! gyp ERR! cwd D:\BaiduNetdiskDownload\数据中台\9-springboot springcloud一站式数据中台 大数据\源码+资料+文档\datax-ui-pro\node_modules\deasync
npm ERR! gyp ERR! node -v v18.17.1
npm ERR! gyp ERR! node-gyp -v v10.0.1
npm ERR! gyp ERR! not ok
npm ERR! Build failed

npm ERR! A complete log of this run can be found in: D:\Download\nodejs\node_cache\_logs\2024-04-05T17_03_33_105Z-debug-0.log

**Comment by digitarald** [comment]
It looks like you have issues with springboot, not extension building
